barretenberg/cpp/src/barretenberg/vm/avm_trace/AvmMini_mem_trace.hpp

```diff
@@ -16,21 +16,21 @@ class AvmMiniMemTraceBuilder {
     static const uint32_t SUB_CLK_STORE_C = 5;
 
     struct MemoryTraceEntry {
```

Comment:
writing comments about `AvmMini_mem_trace.cpp` here as the file isn't in the PR

The method `AvmMiniMemTraceBuilder::compare_mem_entries`, would it be simpler if an `operator::<` method was implemented for `MemoryTraceEntry`? It would accomplish the same thing and mean that `std::sort` can be called on containers of `MemoryTraceEntry` objects, without having to specify `compare_mem_entries` method (just adds a small amount of context overhead if this code is used elsewhere)

Reply:
Excellent idea. I pushed a change to implement your suggestion.